code/modules/research/designs/weapon_designs.dm

```diff
@@ -2,6 +2,60 @@
 /////////////////Weapons/////////////////
 /////////////////////////////////////////
 
+/datum/design/c38/sec
+	name = "Speed Loader (.38)"
+	id = "sec_38"
+	build_type = PROTOLATHE
+	materials = list(MAT_METAL = 20000)
+	build_path = /obj/item/ammo_box/c38
+	category = list("Ammo")
+	departmental_flags = DEPARTMENTAL_FLAG_SECURITY
```

Comment:
Remove 6,7,9,10

Reply:
I can't remove 7, it needs an ID distinct from its parent